client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -9,9 +9,15 @@ import MediaBlock from '../blocks/MediaBlock';
  * Editor block to display media and edit content.
  */
 const EmbedBlock = props => {
-  const { entity, onRemoveEntity } = props.blockProps;
+  const { entity, onEditEntity, onRemoveEntity } = props.blockProps;
   const { url, title, thumbnail } = entity.getData();
 
+  const onBlockEditEntity = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
```

Comment:
I’m a bit surprised we would need this for the Edit button but not for the Remove one. Are those really needed? If so it would be nicer to add `type="button"` to the button, which should hopefully make this irrelevant.

Reply:
Yep, adding `type="button"` did the trick. Thanks for the suggestion. I removed this method as it's no longer required.